cmd/infra/aws/create.go

```diff
@@ -88,42 +107,52 @@ func NewCreateCommand() *cobra.Command {
 	cmd.Flags().StringVar(&opts.OutputFile, "output-file", opts.OutputFile, "Path to file that will contain output information from infra resources (optional)")
 	cmd.Flags().StringVar(&opts.Region, "region", opts.Region, "Region where cluster infra should be created")
 	cmd.Flags().StringSliceVar(&opts.AdditionalTags, "additional-tags", opts.AdditionalTags, "Additional tags to set on AWS resources")
-	cmd.Flags().StringVar(&opts.Name, "name", opts.Name, "A name for the cluster")
+	cmd.Flags().StringVar(&opts.Name, "name", opts.Name, "A name for the cluster that is used as the sub-domain for the ingress url")
 	cmd.Flags().StringVar(&opts.BaseDomain, "base-domain", opts.BaseDomain, "The ingress base domain for the cluster")
 	cmd.Flags().StringSliceVar(&opts.Zones, "zones", opts.Zones, "The availablity zones in which NodePool can be created")
 	cmd.Flags().BoolVar(&opts.EnableProxy, "enable-proxy", opts.EnableProxy, "If a proxy should be set up, rather than allowing direct internet access from the nodes")
+	cmd.Flags().BoolVar(&opts.RenderResource, "render-yaml", opts.RenderResource, "Render a HostedClusterInfrastructure custom resource yaml")
+	cmd.Flags().StringVar(&opts.Namespace, "namespace", opts.Namespace, "Namespace to use when rendering HostedClusterInfrastructure")
```

Comment:
I had to think a little bit to understand what this meant. Maybe "Namespace of the HostedClusterInfrastructure"?

Reply:
Could make it render-hci, or could try to automatically create the HCI is there is a kube connection...

Looking for votes.

Reply:
Went with `--render` per suggestions below.